### PR TITLE
use @ovh-ux/manager-freefax and @ovh-ux/manager-overthebox

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

* update @ovh-ux/manager-core@^1.2.0
* update @ovh-ux/manager-freefax@^1.0.6
* update @ovh-ux/manager-overthebox@^1.1.5
* add @ovh-ux/manager-telecom-styles@^2.0.2
* add .telecom-legacy class
* clean styles (moved in @ovh-ux/manager-telecom-styles)
* update directives usages from @ovh-ux/ng-ovh-contracts
* add serviceName update in sidebar

7a0f7ec0 — refactor: move sidebar and navbar in index.html
a45a13dd — refactor: use @ovh-ux/manager-freefax@^1.0.0-alpha.1
0214e846 — refactor: use @ovh-ux/manager-overthebox@^1.0.0-alpha.1
94233c36 — refactor: update freefax and otb
9e47fc5f — fix(deps): update ovh-angular-contracts
99320228 — refactor: update freefax and otb
a8a99e2d — build(webpack): add mainFields in resolve
822d7c70 — fix(less): update rem values
2e4ea41e — fix(deps): remove @ovh-ux/ovh-angular-contracts dependency

ref: OM-37 && OM-38

### Related

* [x] Add `ws serviceType` config https://github.com/ovh-ux/manager/pull/101

### Todo : 

* [x] Rebase after merging https://github.com/ovh-ux/ovh-manager-telecom/pull/1237

/cc @jleveugle @frenautvh @antleblanc

### CODE REVIEW

File: client/app/app.js [hunk 1]
```diff
@@ -57,6 +61,9 @@ angular.module('managerApp', [
   'angular-translate-loader-pluggable',
   telecomUniverseComponents,
   translateAsyncLoader,
+  managerCore,
```
Comment (antleblanc) [inline client/app/app.js]: Could be sorted?

File: client/app/app.less [hunk 2]
```diff
@@ -31,6 +31,7 @@
 @import "ovh-angular-line-diagnostics/src/ovh-angular-line-diagnostics/ovh-angular-line-diagnostics.less";
 @import "ovh-angular-timeline/src/ovh-angular-timeline.less";
 @import (less) "ovh-ng-input-password/dist/ovh-ng-input-password.css";
+@import "@ovh-ux/manager-universes-styles/src/telecom";
```
Comment (antleblanc) [inline client/app/app.less]: Order is important? If not, could be sorted?

File: client/app/index.js [hunk 3]
```diff
@@ -64,7 +64,7 @@ import 'ovh-angular-swimming-poll';
 import 'ovh-api-services';
 import 'ovh-angular-contact';
 import 'jquery.scrollto';
-import 'ovh-angular-contracts';
+import '@ovh-ux/ovh-angular-contracts';
```
Comment (antleblanc) [inline client/app/index.js]: Could be sorted?

File: package.json [hunk 4]
```diff
@@ -42,6 +42,12 @@
     "@bower_components/crypto": "anodynos/node2web_crypto#3.46.1",
     "@bower_components/international-phone-number": "mareczek/international-phone-number#^0.0.16",
     "@bower_components/matchmedia": "paulirish/matchMedia.js#~0.2.0",
+    "@ovh-ux/angular-uirouter-title": "^1.0.4",
+    "@ovh-ux/manager-core": "^1.0.0-alpha.0",
+    "@ovh-ux/manager-freefax": "^1.0.0-alpha.1",
+    "@ovh-ux/manager-overthebox": "^1.0.0-alpha.1",
+    "@ovh-ux/manager-universes-styles": "^1.0.0-alpha.0",
+    "@ovh-ux/ovh-angular-contracts": "https://github.com/ovh-ux/ovh-angular-contracts.git#v3.x.x",
```
Comment (antleblanc) [inline package.json]: Quick reminder that we have to release this dependency.

Comment (cbourgois) [reply package.json]: Done using `^3.0.0-beta.0`

**Comment by jleveugle** [comment]
Hello @cbourgois,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by ovh-cds** [comment]
CDS Report build-manager#1168.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✘

**Comment by antleblanc** [comment]
b9ebf0c: You misspelled `manager-inverses-styles`.

Furthermore it's a bit redundant with #1237.

You should prefer using this way when importing dependency:

```js
import ngUirouterTitle from '@ovh-ux/ng-uirouter-title';

angular
  .module('myApp', [ngUirouterTitle]);
```

**Comment by ovh-cds** [comment]
CDS Report build-manager#1184.0 ■   
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✔ 
  * Build manager EU ■   
  * Build manager US ✔ 
  * Tests ✘

**Comment by ovh-cds** [comment]
CDS Report build-manager#1185.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✔ 
  * Build manager EU ✔ 
  * Build manager US ✔ 
  * Tests ✘

**Comment by ovh-cds** [comment]
CDS Report build-manager#1185.1 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✘

**Comment by ovh-cds** [comment]
CDS Report build-manager#1220.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✔ 
  * Build manager EU ✘  
  * Build manager US ✔ 
  * Tests ✔

**Pull request merged**